pages/overview/_meta.js

```diff
@@ -16,7 +21,10 @@ export default {
   },
   deployment: "Deployment",
   typescript: "Typescript",
-  "migrating-to-v5": "Migrate to NextAuth@5",
+  "-- More": {
+    type: "separator",
+    title: "More",
+  },
```

Comment:
In theory, we were going to move these two pages, **Contributors** and **Sponsors**, to the footer, so the sidebar for "Getting started" will stay quite lean, hence maybe no need for the sections 💭

Reply:
Yeah i saw that note, added them to the footer too I'm that PR of mine, but it doesn't matter to me. If y'all wanna remove them here, just go for it 👍

Reply:
Ahh haha this is that PR 😅 too many tabs open haha. But yeah feel free to remove them there if y'all want. Balazs had a little commet on slack that maybe we leave them here then too 🤔